From 6ed96b4228b03e3ab159523e30432f66e29d0909 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Mon, 27 Sep 2021 13:30:23 -0400
Subject: [PATCH] Disable blosc2 for conda-forge

---
 setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.py b/setup.py
index 3c8b640..a9a453b 100644
--- a/setup.py
+++ b/setup.py
@@ -407,6 +407,7 @@ def customize_build_cf(EXTENSIONS, OPTIONS):
     del EXTENSIONS['jpeg12']
     del EXTENSIONS['jpegxl']
     del EXTENSIONS['zlibng']
+    del EXTENSIONS['blosc2']
 
     # build the jpeg8 extension against libjpeg v9 instead of libjpeg-turbo
     EXTENSIONS['jpeg8']['cythonize'] = True
-- 
2.33.0

