diff --git a/setup.py b/setup.py
index f0e4f7a..2953b1a 100644
--- a/setup.py
+++ b/setup.py
@@ -527,7 +527,6 @@ def customize_build_condaforge(EXTENSIONS, OPTIONS):
     del EXTENSIONS['sperr']
     del EXTENSIONS['sz3']
     del EXTENSIONS['ultrahdr']
-    del EXTENSIONS['zlibng']
 
     EXTENSIONS['jpeg8']['sources'] = []  # use libjpeg-turbo 3
 
@@ -554,6 +553,7 @@ def customize_build_condaforge(EXTENSIONS, OPTIONS):
         ]
         EXTENSIONS['jpegxr']['libraries'] = ['libjpegxr', 'libjxrglue']
         EXTENSIONS['szip']['libraries'] = ['szip']
+        EXTENSIONS['zlibng']['libraries'] = ['zlib-ng']
     else:
         EXTENSIONS['zopfli']['include_dirs'] = [
             os.path.join(os.environ['PREFIX'], 'include', 'zopfli')
